builtin/game/constants.lua

```diff
@@ -20,3 +20,6 @@ core.EMERGE_GENERATED   = 4
 
 -- constants.h
 core.MAP_BLOCKSIZE = 16
+
+-- Maximum value for node 'light_source' parameter
```

Comment:
It's from light.h, which could be noted here.

ShadowNinja's comment about noting it in the header file as well, can apply here too.
https://github.com/minetest/minetest/commit/c8b4bedaa89ac5246161b1bb9e3826dcce083fcf#commitcomment-18510949